packages/hash/api/src/model/knowledge/page.model.ts

```diff
@@ -249,7 +249,7 @@ export default class extends EntityModel {
   ): Promise<boolean> {
     const { page } = params;
 
-    if (this.entityId === page.entityId) {
+    if (this.entityUuid === page.entityUuid) {
```

Comment:
Should we just compare `baseId` here?

Reply:
Addressed as part of https://github.com/hashintel/hash/commit/3cb6ed29868d750d40108e418521ade0b2bf056f